### ISSUE TITLE
Style only loaded after browser refresh

### ISSUE THREAD

**Issue by go4st** [appreciated/vaadin-app-layout#297]
I use the vaadin-app-layout within a Spring Boot Vaadin 14 (NPM) application. The styles for the app layout are not loaded when i enter the page, and following warning appears in the chrome console: 

`VM4974:339 Resource interpreted as Stylesheet but transferred with MIME type text/html: "http://localhost:8080/frontend-es6/com/github/appreciated/app-layout/app-layout-styles-lumo.css".

VM4974:339 Resource interpreted as Stylesheet but transferred with MIME type text/html: "http://localhost:8080/frontend-es6/com/github/appreciated/app-layout/left/left-responsive-hybrid-no-app-bar.css".`

After page refresh the warning disappears and styles are loaded correctly.

**Comment by appreciated** [comment]
Can you provide more context please? Are you using multiple behaviours? This sounds more like a Vaadin bug to me.

**Comment by go4st** [comment]
I found the issue and yes its not a bug from the addon. I used `UI.getCurrent().navigate(...)` in my login view to navigate to the main ui after login. Inspecting Google Chromes network tab showed, that the styles of the addon are loaded after the DOM loading is already done. I could fix that behaviour by using `UI.getCurrent().getPage().setLocation(...)` but I'm not sure if that is the desired usage of Vaadin's navigation. Thank you for the quick reply!

**Comment by appreciated** [comment]
What I recommend doing is creating minimal (non-)working example of the bug and create an issue on the [flow repository](https://github.com/vaadin/flow/issues) and appending the example there, to fix the issue for good.

**Comment by maksimu** [comment]
I'm encountering the same issue on my side. Only after refreshing  styles are loaded

![image](https://user-images.githubusercontent.com/187306/67504863-08a64e00-f63f-11e9-9acf-6d7a3cb9b474.png)

**Comment by maksimu** [comment]
Also to note, make sure you are redirecting to the existing route. I was redirecting to the  empty one, but once I changed it to "/" then everything started to work fine again

**Comment by urkl** [comment]
We have the same problem.

**Comment by luxingxiao** [comment]
We have the same problem.

**Comment by appreciated** [comment]
Can you confirm that this issue was fixed in [Vaadin 14.2.1](https://github.com/vaadin/platform/releases/tag/14.2.1)? The related issue seems to be fixed in the included [`flow` version 2.2.1](https://github.com/vaadin/flow/releases/tag/2.2.1)

**Comment by Dudeplayz** [comment]
It seems to be fixed. Tested with 14.2.3.

**Comment by appreciated** [comment]
@Dudeplayz thanks for the feedback!
